### ISSUE TITLE
A11y_Fluent UI_Components_Text spacing: After apply text spacing under components "Acordion, Alert, Avatar" links are not visible clearly.

### ISSUE THREAD

**Issue by Latha2564** [microsoft/vscode-azurefunctions#3393]
**GitHubTags:**

 #A11ySev2; #TCS-E+d; #NFT-Fluentui-Web-Oct2022; #DesktopWeb; #WCAG1.4.12.
 #Text spacing; #A11yMAS.

**OS Details:** 

 Web:win11

**Environment Details:**

 Application Name: Fluent Ui

**Repro Steps:**

1.Hit enter on the URL https://react-dev.fluentui.dev/
2."Fluent UI React v9" page will open.
3.Tab till "Components" button and hit enter.
4.Tab till "Accordion" link on the left side bar and hit enter. 
5.Apply text spacing
6.verify under components "accordion,avatar.alert" links are not visible

**Actual Result:**
  
After apply text spacing under components "Accordion, Alert, Avatar" links are not visible clearly

**Expected Result:**

When we apply text spacing under components "Acordion, Alert, Avatar" links should be visible clearly

**User Impact:**

Text spacing users will get difficulties when "Accordion, Alert, Avatar" links are not visible clearly

**Comment by Latha2564** [comment]
Closing this bug as the bug is ogged in wrong ADO
